scripts/time_plot.py

```diff
@@ -3229,23 +3292,25 @@ def pack_binarydata_msg(varname,val,dtype):
     return buff
 
 #Caught exception (local variable 'action' referenced before assignment). Removing registered handler
-def parse_websock_cmd(s, request):
+def parse_websock_cmd(s, handlerkey):
```

Comment:
@bmerry just do a final squiz from here onwards as these changes have widespread affects whereas the changes above are only relevant to the issuing of the 'holo' command.